account_banking_payment_export/README.rst

```diff
@@ -76,6 +76,7 @@ Contributors
 * Erwin van der Ploeg
 * Raphaël Valyi
 * Sandy Carter
+* Angel Moya <angel.moya@domatix.com>
```

Comment:
You should put the credits in the change module (_account_payment_partner_, not here).